Tests/powershell-yaml.Tests.ps1

```diff
@@ -94,6 +94,17 @@ InModuleScope $moduleName {
         }
 
         Context "Test array handling under various circumstances." {
+
+            It "Should ConvertTo empty array." {
+                $res = ConvertTo-Yaml @()
+                $res.Trim() | Should -Be "[]"
+            }
+
+            It "Should ConvertFrom empty array." {
+                $res = "[]" | ConvertFrom-Yaml
+                $res | Should -Be @()
```

Comment:
This will always fail. An empty array piped to this commandlet will always return ```$null```. See previous comment about why.

Reply:
Thanks for pointing that out! That's still confusingly unexpected behavior to me.